src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs

```diff
@@ -831,22 +818,10 @@ private static uint HandleEventPeerRecvAborted(State state, ref StreamEvent evt)
             return MsQuicStatusCodes.Success;
         }
 
-        private static uint HandleEventStartComplete(State state)
+        private static uint HandleEventStartComplete(State state, ref StreamEvent evt)
```

Comment:
Do we need to pass in the event if we are not using it? (now?)

Reply:
We don't need to. I added it because I thought I was going to use it, but since we don't have a definition for the START_COMPLETE event data, we can't right now. I don't think there's any harm in having it, though.